examples/pylab_examples/pythonic_matplotlib.py

```diff
@@ -36,19 +36,27 @@
 
 The syntax of set is
 
+.. code::
```

Comment:
ahh I see what I was getting wrong before - I wasn't adding the `::` at the end of paragraphs. will switch this back, agree it's cleaner